plugins/utils/wilder-nvim.nix

```diff
@@ -22,7 +22,7 @@ in {
 
       package = helpers.mkPackageOption "wilder-nvim" pkgs.vimPlugins.wilder-nvim;
 
-      modes = helpers.defaultNullOpts.mkNullable (types.listOf types.str) "['/', '?']" ''
+      modes = helpers.defaultNullOpts.mkNullable (types.listOf types.str) "[\"/\", \"?\"]" ''
```

Comment:
There are no `,` in nix lists

Reply:
Also you can use `''["/" "?"]''` to avoid escaping

Reply:
Thanks, Python habits die hard :smile: